recipes/glslang/all/conanfile.py

```diff
@@ -0,0 +1,132 @@
+            self.requires.add("spirv-tools/2020.1")
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename(self.name + "-" + self.version, self._source_subfolder)
+
+    def build(self):
+        self._patches_sources()
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def _patches_sources(self):
+        # Do not force PIC
+        cmake_files_to_fix = [
+            {"target": "OGLCompiler", "relpath": os.path.join("OGLCompilersDLL", "CMakeLists.txt")},
+            {"target": "SPIRV"      , "relpath": os.path.join("SPIRV", "CMakeLists.txt")},
+            {"target": "SPVRemapper", "relpath": os.path.join("SPIRV", "CMakeLists.txt")},
+            {"target": "glslang"    , "relpath": os.path.join("glslang", "CMakeLists.txt")},
+            {"target": "OSDependent", "relpath": os.path.join("glslang", "OSDependent", "Unix","CMakeLists.txt")},
+            {"target": "OSDependent", "relpath": os.path.join("glslang", "OSDependent", "Windows","CMakeLists.txt")},
+            {"target": "HLSL"       , "relpath": os.path.join("hlsl", "CMakeLists.txt")},
+        ]
+        for cmake_file in cmake_files_to_fix:
+            tools.replace_in_file(os.path.join(self._source_subfolder, cmake_file["relpath"]),
+                                  "set_property(TARGET {} PROPERTY POSITION_INDEPENDENT_CODE ON)".format(cmake_file["target"]),
+                                  "")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_EXTERNAL"] = False
+        self._cmake.definitions["SKIP_GLSLANG_INSTALL"] = False
+        self._cmake.definitions["ENABLE_SPVREMAPPER"] = self.options.SPVRemapper
+        self._cmake.definitions["ENABLE_GLSLANG_BINARIES"] = self.options.build_executables
+        self._cmake.definitions["ENABLE_GLSLANG_WEB"] = False
+        self._cmake.definitions["ENABLE_GLSLANG_WEB_DEVEL"] = False
+        self._cmake.definitions["ENABLE_EMSCRIPTEN_SINGLE_FILE"] = False
+        self._cmake.definitions["ENABLE_EMSCRIPTEN_ENVIRONMENT_NODE"] = False
+        self._cmake.definitions["ENABLE_HLSL"] = self.options.hlsl
+        self._cmake.definitions["ENABLE_OPT"] = self.options.with_spirv_tools
+        self._cmake.definitions["ENABLE_PCH"] = True
+        self._cmake.definitions["ENABLE_CTEST"] = False
+        self._cmake.definitions["USE_CCACHE"] = False
+        self._cmake.configure(build_folder=self._build_subfolder)
+        return self._cmake
+
+    def package(self):
+        self.copy("LICENSE.txt", dst="licenses", src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
+        tools.rmdir(os.path.join(self.package_folder, "lib", "cmake"))
+
+    def package_info(self):
+        # CMake Targets: SPIRV, glslang, OGLCompiler, OSDependent, SPVRemapper, HLSL
+        # To update when conan components available
```

Comment:
```suggestion
        # TODO: update when conan components available
```

We can filter TODO in the future

Reply:
Ok, I'll update the PR once `spirv-tools` is merged (therefore `with_spirv_tools` default value will be True, since it is preferred when hlsl is True).